### PR TITLE
dbeaver/pro#1640 update error message

### PR DESCRIPTION
**Pull request by Destrolaric**
Closes dbeaver/pro#1640
image

### CODE REVIEW

File: plugins/org.jkiss.dbeaver.registry/src/org/jkiss/dbeaver/registry/driver/MissingDataSourceProvider.java [hunk 1]
```diff
@@ -53,7 +53,9 @@ public DBPPropertyDescriptor[] getConnectionProperties(DBRProgressMonitor monito
     @NotNull
     @Override
     public DBPDataSource openDataSource(@NotNull DBRProgressMonitor monitor, @NotNull DBPDataSourceContainer container) throws DBException {
-        throw new DBException("Corrupted datasource configuration. Driver '" + container.getDriver().getFullName() + "' not found.");
+        throw new DBException("Error: Invalid Datasource Configuration\n"
```
Comment (serge-rider) [inline plugins/org.jkiss.dbeaver.registry/src/org/jkiss/dbeaver/registry/driver/MissingDataSourceProvider.java]: Let's not use mixed case in error messages.

**Review by serge-rider — CHANGES_REQUESTED** [review]

**Review by LonwoLonwo — APPROVED** [review]

**Comment by MashaKorax** [comment]
Please, add the period to separate these sentences

dbeaver_Z4aKPp839H

**Comment by E1izabeth** [comment]
Do we really need to repeat `Error: invalid datasource configuration` as it's already presented in the final message of the dialog?

**Comment by MashaKorax** [comment]
verified

**Pull request merged**